requirements.txt

```diff
@@ -6,3 +6,4 @@ numpy  # train.py dataset memmap
 jsonargparse[signatures]  # generate.py, convert_checkpoint.py CLI
 bitsandbytes  # quantization.py
 datasets  # evaluate.py
+git+https://github.com/huggingface/transformers
```

Comment:
I'd avoid adding transformers to the dependencies since it's only used during verification, and instead produce a useful error message ("verification requires transformers to be installed, please pip install ..." during verification if you don't have it.